docs/central-entities.rst

```diff
@@ -57,11 +64,9 @@ What's available now:
 
```

Comment:
I'm reading these docs top down (after looking at the individual commits) and I feel like the upper list of "what's available now" could also use a bullet point or two about updating entities and offline entities. I'm not sure if "Use Entities in one or more follow-up form" covers it.